### PR TITLE
🆕(build) Add home/landing featured story logic

### PR DESCRIPTION
**Pull request by Snugug**
* Codifies logic for the featured stories on the home and landing pages (available in `landingCollections.home` and `landingCollections.landing` respectively).
* Makes stories sorted by date as opposed to title and weight
* Corrects the section for a blog post that was missed with a previous PR
* Logo fix

There should be very little visual difference in the staging site for this PR

### CODE REVIEW

**Review by nickalcantara — CHANGES_REQUESTED** [review]
LGTM, just one small call out for a function description that I think may have been deleted on accident.

From what I can tell, it seems like this allows for up to 3 featured stories on the homepage and ensures that the featured story on the landing page is _not_ one that's been featured on the homepage, which makes sense to me!

File: site/_data/eleventyComputed.js [hunk 1]
```diff
@@ -16,7 +16,58 @@
...
+  const home = [];
+  let landing = content[0].data.featured;
+
+  const homeTags = [];
+  const homeFiles = [];
+
+  for (const c of content) {
+    const type = get(c, 'data.tags[1]');
+    if (type) {
+      if (home < 3) {
```
Comment (nickalcantara) [inline site/_data/eleventyComputed.js:38]: Should this be `home.length < 3?`

**Review by nickalcantara — DISMISSED** [review]

File: site/_data/eleventyComputed.js (hunk 1 above)
Comment (Snugug) [reply site/_data/eleventyComputed.js]: Yes. Updated.

**Review by nickalcantara — DISMISSED** [review]

**Review by Vanss472 — APPROVED** [review]
LGTM!

**Pull request merged**